test/externalTests/ens.sh

```diff
@@ -69,9 +69,9 @@ function ens_test
     neutralize_packaged_contracts
 
     # In some cases Hardhat does not detect revert reasons properly via IR.
-    # TODO: Remove this when https://github.com/NomicFoundation/hardhat/issues/2115 gets fixed.
+    # TODO: Remove this when https://github.com/NomicFoundation/hardhat/issues/3365 gets fixed.
     sed -i "s|it\(('Does not allow wrapping a name you do not own',\)|it.skip\1|g" test/wrapper/NameWrapper.js
-    # TODO: Remove this when https://github.com/NomicFoundation/hardhat/issues/2453 gets fixed.
+    # TODO: Remove this when https://github.com/NomicFoundation/hardhat/issues/3365 gets fixed.
```

Comment:
We could just merge with the section above.
```suggestion
```

Reply:
Good point, didn't even notice. `%s/XXXX/3365/g` goes brrr.